server/testdriver/core/src/main/java/org/infinispan/server/test/api/HotRodTestClientDriver.java

```diff
@@ -69,6 +69,16 @@ public <K, V> RemoteCache<K, V> get() {
       return remoteCacheManager.getCache(name);
    }
 
+   /**
+    * Gets a cache with the provided name
+    *
+    * @return {@link RemoteCache}, the cache is such exist
+    */
+   public <K, V> RemoteCache<K, V> get(String name) {
+      RemoteCacheManager remoteCacheManager = testClient.registerResource(testServer.newHotRodClient(clientConfiguration));
+      return remoteCacheManager.getCache(name);
```

Comment:
I think this and the method above are incorrect. It should be
```suggestion
      return createRemoteCacheManager().getCache(name);
```
otherwise, the `port` is ignored.